heartbeat/monitors/active/http/http_test.go

```diff
@@ -626,7 +626,9 @@ func TestNewRoundTripper(t *testing.T) {
 			require.NotNil(t, transp.Dial)
 			require.NotNil(t, transport.TLSDialer)
 
-			require.Equal(t, (&tlscommon.TLSConfig{}).ToConfig(), transp.TLSClientConfig)
+			expected := (&tlscommon.TLSConfig{}).ToConfig()
+			require.Equal(t, expected.InsecureSkipVerify, transp.TLSClientConfig.InsecureSkipVerify)
+			require.NotNil(t, transp.TLSClientConfig.VerifyConnection)
```

Comment:
Is there a chance we will have to remove these two lines in the future? If so lets add a comment why these are checked here, so future "us" know it is ok to remove the check if we change things.

Reply:
When we remove the common name support from libbeat, the second line has to be removed. In general, I did not find this test to be useful at all, but I did my best to keep it.